x-pack/plugins/ml/public/application/components/help_popover/help_popover.scss

```diff
@@ -0,0 +1,13 @@
+.mlHelpPopover__panel {
+  max-width: $euiSize * 30;
+}
+
+.mlHelpPopover__content {
+  @include euiYScrollWithShadows;
+  max-height: 40vh;
+  padding: $euiSizeM;
+}
+
+.mlHelpPopover__buttonIcon {
+  margin-right: $euiSizeXS;
+}
```

Comment:
Is this right margin style necessary? If not, I suggest removing.

Reply:
It doesn't seem to have made a noticeable difference, so I removed it in 6e20e6710577f1bb51a87830e22e8ce01774e40a